installer/install.py

```diff
@@ -0,0 +1,70 @@
+import os
+import sys
+import shutil
+import zipfile
+import urllib.request
+
+
+def __UpdateMissionScriptingLua(path):

```

Comment:
I may be missing something but won't this add the GRPC line every time the program is run? (e.g. someone using it to update versions)